README.md

```diff
@@ -1,2 +1,57 @@
 # strike-off-objections-api
+
+### Overview
 API for handling objections to a company being struck off the register
+
+### Requirements
+
+In order to run the service locally you will need the following:
+
+- [Java 8](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+- [Maven](https://maven.apache.org/download.cgi)
+- [Git](https://git-scm.com/downloads)
+- [MongoDB](https://www.mongodb.com/)
```

Comment:
You don't need mongoDB installed locally to run the app, docker will spin it up from a pre made container